### DISCUSSION TITLE
ESP not responding, need a hand

### DISCUSSION THREAD

**Discussion by Mrkricfalusi** [emporia-vue-local/esphome#43] [Q&A]
I just flashed my Vue and everything was working great. worked great with home assistant. Uploaded new firmware about a dozen times to get everything configured correctly, everything was cake.

Lastly I went to change the name of the device via the second line "name: emporiavue2' to "name: emporia_vue_2", changed it hit save, and ran "esphome run emporia.yaml" which it errored out because it could not find emporia_vue_2.local which makes a world sense. since the device was originally empriavue2.local. I then "esphome run .\emporia.yaml --device 192.168.1.216" and nothing could not find a device, Went unplugged/replug in the Vue and now it is now where to be seen on my network :(.

I removed the Vue from my panel hooked it back up to my little HiLetgo CP2102 just the same as I did during the initial flash. The green light on the Vue turns on, but nothing over serial. "esptool flash_id" comes up with nothing. Tired powering via 5v pin then the 3v pin and nothing happens. Checked my ground pin and GPIO0 and RX/TX were hooked up right. nothing. So I am at a loss as what to do now. 

Any suggestions would be GREATLY apricated. 

I read that sometimes the cheap little USB adapters sometimes do not have enough power and brown out. I used the 5v pin from another ESP32 device and shared grounds using only RX,TX and Ground with the adapter. Same thing happens. Just a green light on the device but nothing coming over serial.

**Comment by flaviut** [comment]
When you hook up the serial port with the Vue in normal (not bootloader) mode, can you see any output? You'll want some sort of serial port software, and the baud rate is 115200. Procedure would be to connect the RX, TX, GND first, open the serial console, and then connect power.

When I'm having trouble, I sometimes flip my RX/TX even when I'm sure I have it right.

It doesn't sound like you're having brownout trouble. It's generally possible to boot off the UART adapter, at least for a long enough to get some decent log messages, before the power hungry WiFi turns on.

I'd also like to mention that I haven't had reports of anyone bricking their device before, so I'm confident this will all work out :)

**Comment by Mrkricfalusi** [comment]
I have tried. I have a new ttl adapter coming in i'll see if that helps. Thank you so much for your help!

**Comment by Mrkricfalusi** [comment]
yeah its completely unresponsive. Tried about everything, two different adapters, rx tx pins, 3v/5v pins. metered to see if anything was shorted. its just unresponsive. I guess the next step is trying to replace the SOC.

**Reply by flaviut** [reply]
What a PITA.

Replacing these modules is definitely not easy, and it's so strange that it failed like that. Sorry about the bad luck :(

**Comment by SenexCrenshaw** [comment]
I had a similar problem. I had to hold both EN and IO0 to the top of the ESP (GND). After a second remove just the EN and then after another second, remove the IO0. This booted it into loader mode. I was able to pull the bin off at 115200 baud.

Hopefully this helps.

**Comment by Mrkricfalusi** [comment]
Well, I had hope, got home and first thing tried out the EN pin.  But only thing it really did was make the green LED on the board a swigged dimmer or brighter depending if it was plugged in.. 
So is there anyway to find out of my ESP32 chip is just trashed? Like seeing if a pin goes high or low when booting kinda thing? I am not the best with troubleshooting ESPs. I just just throw a ESPhome yaml together here and there for some simple sensors and controllers. 

 but SenexCrenshaw thank you for your help!

**Reply by SenexCrenshaw** [reply]
I found a few things.
1. The TTL Serial adapter has to be able to power the esp32. I have had some brownouts and problems a few. Make sure you have a quality adapter. Pin 5v goes to the pin all the way to the left of the IO0 pin. If you put in on the 3.3 then it might be trashed
2. Try reversing RX and TX
3. Hold both the IO0 (boot) and EN, then release EN and then release the BOOT. EN reboot it and pulling IO0 low will cause it to boot into firmware/flash mode.
4. Try 9600 and 115200 for baud rates

Good luck!

**Comment by Mrkricfalusi** [comment]
it's burn't. Ordered another one. I'm good now
